code/game/gamemodes/cult/runes.dm

```diff
@@ -379,13 +380,16 @@ structure_check() searches for nearby cultist structures required for the invoca
 		if(sacrifice_fulfilled)
 			to_chat(M, "<span class='cultlarge'>\"Yes! This is the one I desire! You have done well.\"</span>")
 		else
-			if(ishuman(offering) || isrobot(offering))
+			if(ishuman(offering) && offering.mind.offstation_role && offering.mind.special_role != SPECIAL_ROLE_ERT ) //If you try it on a ghost role, you get nothing
+				to_chat(M, "<span class='cultlarge'>\"This soul is worthless to you and I.\"</span>")
```

Comment:
```suggestion
				to_chat(M, "<span class='cultlarge'>\"This soul is worthless to me.\"</span>")
```

So looking over steels review, I'd say the second option he suggested was the best one. You and I sounds oddly formal here.

Reply:
Trying to convey that the cult is unable to use the soul either, which is why they do not get a soulstone with the soul in it, so thats why the god and you are mentioned

Reply:
Then perhaps "This soul is of no use to either of us"